package-lock.json

```diff
@@ -2660,53 +2660,6 @@
       "integrity": "sha512-6U71C2Wp7r5XtFtQzYrW5iKFT67OixrSxjI4MptCHzdSVlgabczzqLe0ZSgnub/5Kp4hSbpDB1tMytZY9pwxxA==",
```

Comment:
I think some of these changes are incorrect (e.g. it looks like eslint got removed from the dependency list)-- probably the right thing to do here is just to revert the changes to both `package-lock.json` and `package.json` and re-add the dependency using `npm add`